recipes/cccl-python/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "cccl-python" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cccl/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3ded63682f94071685cd6492cee70ad2a1feae02abfb3fbcce50e2d2f22edf4e
+
+build:
+  script: |
+    cd python/cuda_cccl
+    {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+  host:
+    - python
+    - cython
+    - scikit-build-core >=0.10
+    - setuptools-scm
+    - pip
+  run:
+    - python
+    - numba >=0.60.0
+    - cuda-pathfinder >=1.1.0
+    - cuda-core
+    - numba-cuda >=0.18.0
+
+test:
+  imports:
+    - cuda.cccl
+    - cuda.cccl.cooperative
+    - cuda.cccl.parallel
+
+about:
+  home: https://github.com/NVIDIA/cccl
+  summary: 'CUDA Core Library for Python'
+  description: 'CUDA Core Library for Python'
+  license: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+  license_file: LICENSE
+  # The doc_url and dev_url are optional.
+  doc_url: https://simplejson.readthedocs.io/
+  dev_url: https://github.com/simplejson/simplejson
```

Just didn't want to forget to change these.
```suggestion
  doc_url: https://nvidia.github.io/cccl/python/index.html
  dev_url: https://github.com/NVIDIA/cccl
```

---

Much appreciated 😮‍💨